ideascube/conf/kb_fra_crfaix.py

```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # pragma: no flakes
+
+LANGUAGE_CODE = 'fr'
+IDEASCUBE_NAME = 'CRF AIX'
```

Comment:
They are not really necessary actually, but because I can specify variable I did it...